notification/twilio/alertsms.go

```diff
@@ -89,14 +89,34 @@ func mapGSM(r rune) rune {
 	return '?'
 }
 
+func oneOfPrefixes(s string, prefixes ...string) bool {
```

Comment:
Dunno how y'all feel about unit testing un-exported functions, but you could add a quick unit test for this

Reply:
https://github.com/target/goalert/blob/master/CONTRIBUTING.md#testing

As a general rule, we add a unit test if:
- The function is too complex to validate by visually looking at the code
- We need to benchmark for performance and/or memory concerns in a hot path
- We're writing an Example test for a constructor or exported API method
- The functionality is used from several call-sites (esp. if exported)
- There are edge cases that a provable table test would aid in review for
- Proving behavior for/during refactoring.


Something like this wouldn't have much value to be unit tested, other than increasing the number of unit tests, which would just add noise. This is a throwaway function to be used a few lines from its definition. It could just as easily live inside the two functions using it.

I did add a smoke test to functionally validate the system behavior, which would survive a refactor -- but in said refactor, these functions would all likely be deleted to use some other approach (like an API call).

That being said, a better name & comment would go a long way towards that visual validation, so I updated it.

Reply:
Sweet, thanks for the info!